Dockerfile

```diff
@@ -32,6 +32,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-
     libnuma-dev \
     miopen-hip \
     rocblas \
+    half \
```

Comment:
funny, I tried adding this before but was reminded that "half" gets installed via line 53 in the docker because it is listed in requirements.txt.  So it shouldn't be needed here.

Reply:
Done